client/lib/application/src/version-list/application-version.hbs

```diff
@@ -0,0 +1,13 @@
+<h2>{{app.name}} Versions</h2>
+
+{{#if versions.length}}
+    <ul>
+        {{#each versions}}
+            <li>
+                <a href="version/{{this.id}}">{{this.id}}</a>
```

Comment:
Thing is that we deploy quite often currently for different reasons. And when smth doesn’t work, people will complain about it.

Maybe we should add a big [beta stamp](http://www.mijndoelenstellen.nl/wp-content/uploads/2013/12/beta-stamp.png) as background-image.